src/app/app-routing.module.ts

```diff
@@ -4,8 +4,11 @@ import { RacesComponent } from './races/races.component';
 import { AbilitiesComponent } from './abilities/abilities.component';
 import { ClassesComponent } from './classes/classes.component';
 import { SettingsComponent } from './settings/settings.component';
+import { HomeComponent } from './home/home.component';
 
 const routes: Routes = [
+  { path: '', redirectTo: '/home', pathMatch: 'full'},
```

Comment:
You missed the spaces at the end of the line, before the }

Reply:
Adjusted spacing at the end